content/docs/user-guide/gems/reference/multiplayer/multiplayer/tutorial_multiplayer.md

```diff
@@ -0,0 +1,887 @@
+    }
+```
+
+A controller only exists on the authority server, thus this effectively splits the execution into server-only and client-only logic. 
+
+Our full code for the component so far is as follows. The header is `D:\git\o3de-multiplayersample\Gem\Code\Source\Components\MyFirstNetworkComponent.h`:
+
+```c++
+#pragma once
+
+#include <Source/AutoGen/MyFirstNetworkComponent.AutoComponent.h>
+#include <AzCore/Component/TickBus.h>
+
+namespace MultiplayerSample
+{
+    class MyFirstNetworkComponent
+        : public MyFirstNetworkComponentBase
+        , public AZ::TickBus::Handler
+    {
+    public:
+        AZ_MULTIPLAYER_COMPONENT(MultiplayerSample::MyFirstNetworkComponent, s_myFirstNetworkComponentConcreteUuid, MultiplayerSample::MyFirstNetworkComponentBase);
+
+        static void Reflect(AZ::ReflectContext* context);
+
+        void OnInit() override;
+        void OnActivate(Multiplayer::EntityIsMigrating entityIsMigrating) override;
+        void OnDeactivate(Multiplayer::EntityIsMigrating entityIsMigrating) override;
+
+        void OnTick(float deltaTime, AZ::ScriptTimePoint time) override;
+    };
+}
+```
+
+And the source, `D:\git\o3de-multiplayersample\Gem\Code\Source\Components\MyFirstNetworkComponent.cpp`:
+
+```c++
+#include <Source/Components/MyFirstNetworkComponent.h>
+
+#include <AzCore/Serialization/SerializeContext.h>
+
+namespace MultiplayerSample
+{
+    void MyFirstNetworkComponent::Reflect( AZ::ReflectContext* context )
+    {
+        AZ::SerializeContext* serializeContext = azrtti_cast<AZ::SerializeContext*>( context );
+        if ( serializeContext )
+        {
+            serializeContext->Class<MyFirstNetworkComponent, MyFirstNetworkComponentBase>()
+                ->Version( 1 );
+        }
+        MyFirstNetworkComponentBase::Reflect( context );
+    }
+
+    void MyFirstNetworkComponent::OnInit()
+    {
+    }
+
+    void MyFirstNetworkComponent::OnActivate( [[maybe_unused]] Multiplayer::EntityIsMigrating entityIsMigrating )
+    {
+        AZ::TickBus::Handler::BusConnect();
+    }
+
+    void MyFirstNetworkComponent::OnDeactivate( [[maybe_unused]] Multiplayer::EntityIsMigrating entityIsMigrating )
+    {
+        AZ::TickBus::Handler::BusDisconnect();
+    }
+
+    void MyFirstNetworkComponent::OnTick( [[maybe_unused]] float deltaTime, AZ::ScriptTimePoint time )
+    {
+        if ( HasController() )
+        {
+            auto* controller = static_cast<MyFirstNetworkComponentController*>( GetController() );
+            controller->ModifyUpTime() = time.GetSeconds();
+            AZ_Printf( "MyFirstNetworkComponent", "server = %f", GetUpTime() );
+        }
+        else
+        {
+            AZ_Printf( "MyFirstNetworkComponent", "client = %f", GetUpTime() );
+        }
+    }
+}
+```
+
+### Listen for Network Property Changes on Clients
+
+However, let's improve on this. A client has no need to tick, it can instead listen for changes. In order to do that, we need to enable `GenerateEventBindings`.
```

Comment:
nice! get things working, and then slowly introduce new concepts that help build even better code.